README.md

```diff
@@ -160,14 +160,14 @@ Currently **officially** using Airflow:
 1. [Change.org](https://www.change.org) [[@change](https://github.com/change), [@vijaykramesh](https://github.com/vijaykramesh)]
 1. [Chartboost](https://www.chartboost.com) [[@cgelman](https://github.com/cgelman) & [@dclubb](https://github.com/dclubb)]
 1. [Checkr](https://checkr.com) [[@tongboh](https://github.com/tongboh)]
-1. [Children's Hospital of Philadelphia Division of Genomic Diagnostics](http://www.chop.edu/centers-programs/division-genomic-diagnostics) [[@genomics-geek]](https://github.com/genomics-geek/)
+1. [Children's Hospital of Philadelphia Division of Genomic Diagnostics](http://www.chop.edu/centers-programs/division-genomic-diagnostics) [[@genomics-geek]](https://github.com/genomics-geek/)]
```

Comment:
I have doubts whether this change is certainly correct. Could you justify the need for this change?

I add a screenshot to help you find the error
<img width="903" alt="Screenshot 2019-03-30 at 00 37 17" src="https://user-images.githubusercontent.com/12058428/55267738-67ac0c00-5284-11e9-8585-eae32f2cf3fe.png">

Reply:
Thank you for showing how to achieve the perfection of the Air-Flow project. I think that you judge it wrong and you look at it in a bad way, because the total number of errors in the project, thanks to my pull requests, has dropped. Of course, I have not solved all problems, but it is not possible to bring the world to perfection in a short time.

I think that achieving the perfection of the Air-Flow project can be achieved in a larger number of pull requests of more people.